Tests/NIOSSLTests/UnwrappingTests.swift

```diff
@@ -16,8 +16,10 @@ import XCTest
 import NIO
 @testable import NIOSSL
 
+private let tmpDir = FileManager.default.temporaryDirectory.path
+
 func connectInMemory(client: EmbeddedChannel, server: EmbeddedChannel) throws {
-    let addr = try assertNoThrowWithValue(SocketAddress(unixDomainSocketPath: "/tmp/whatever2"))
+    let addr = try assertNoThrowWithValue(SocketAddress(unixDomainSocketPath: "\(tmpDir)/whatever2"))
```

Comment:
We shouldn't need the changes in this file.

Reply:
Yeah, I saw some tests that call `/tmp` failing on Android, so I search-and-replaced all of them. These two work fine the way they were, so I reverted in the latest push.